src/pages/StocktakeEditPage.js

```diff
@@ -161,7 +161,7 @@ export class StocktakeEditPage extends React.Component {
     const { database } = this.props;
     const { usesReasons } = this.state;
 
-    if (key !== 'countedTotalQuantity' || newValue === '') return;
+    if (key !== 'countedTotalQuantity' || !newValue) return;
```

Comment:
There should probably be a comment here explaining this logic and that it is a temporary workaround.